components/src/menu/menu-item.component.ts

```diff
@@ -3,13 +3,16 @@ import { Component, ElementRef } from "@angular/core";
 
 @Component({
   selector: "[bit-menu-item]",
-  templateUrl: "./menu-item.component.html",
   host: {
     class:
-      "tw-block tw-py-1 tw-px-6 !tw-text-main !tw-no-underline hover:tw-bg-secondary-100 focus:tw-bg-secondary-100 focus:tw-z-50 tw-cursor-pointer tw-border-none tw-bg-background tw-text-left",
+      "tw-block tw-py-1 tw-px-6 !tw-text-main !tw-no-underline tw-cursor-pointer tw-border-none tw-bg-background tw-text-left " +
+      "hover:tw-bg-secondary-100 " +
+      "focus:tw-bg-secondary-100 focus:tw-z-50 focus:tw-outline-none focus:tw-ring focus:tw-ring-offset-2 focus:tw-ring-primary-700 " +
+      "active:!tw-ring-0 active:!tw-ring-offset-0",
```

Comment:
Not a huge fan of `+`, thoughts on using an array with join here? A line per class.

Reply:
I tried that, but it wasn't being evaluated properly for some reason. It turns out that `@HostBinding` is the [suggested style anyway](https://angular.io/guide/styleguide#hostlistenerhostbinding-decorators-versus-host-metadata), and it's what you've used elsewhere, so I've refactored to use that with an array instead.